wrangler-service/src/main/java/co/cask/wrangler/service/gcs/GCSService.java

```diff
@@ -275,12 +280,19 @@ public void run(DatasetContext datasetContext) throws Exception {
           object.addProperty("wrangle", isWrangeable);
         }
         values.add(object);
+        if (values.size() >= bucketLimit) {
```

Comment:
@albertshau should we use a separate query param for listing data in a bucket?

Reply:
no i think its fine to have a single limit, as it's a single API.